utils/camera.js

```diff
@@ -9,41 +22,57 @@ module.exports = {
     }
 
     const shakeTimer = game.time.create(false);
+    const shakeInterval = options.randomizeInterval? 
+      Math.random() * options.shakeInterval + options.shakeInterval : 
+      options.shakeInterval;
```

Comment:
Take a look at `Phaser.Math.integerInRange` for a more readable way to express this.